src/main/java/uk/gov/hmcts/reform/prl/services/ServiceOfApplicationService.java

```diff
@@ -462,46 +466,161 @@ private String handleNotificationsForCitizenCreatedCase(CaseData caseData, Strin
+                                               ),
+                                               Templates.PRL_LET_ENG_AP1
+                    );
+                }
+                generateUnservedRespondentPackDaCbCa(caseData, authorization, staticDocs, caseDataMap,
+                                                     caseData.getServiceOfApplication().getSoaCitizenServingRespondentsOptionsDA()
+                                                         .toString());
             }
         }
         return whoIsResponsibleForServing;
     }
 
+    private String handleNotificationsDaPersonalApplicantLip(CaseData caseData, String authorization,
+                                                             List<Element<EmailNotificationDetails>> emailNotificationDetails,
+                                                             List<Element<BulkPrintDetails>> bulkPrintDetails,
+                                                             Map<String, Object> caseDataMap,
+                                                             List<Document> c100StaticDocs) {
+        Element<PartyDetails> applicant = element(caseData.getApplicantsFL401().getPartyId(), caseData.getApplicantsFL401());
+        Element<PartyDetails> respondent = element(caseData.getRespondentsFL401().getPartyId(), caseData.getRespondentsFL401());
+        List<Document> docs = new ArrayList<>();
+        List<Document> packEdocs = getNotificationPack(caseData, PrlAppsConstants.E, c100StaticDocs);
+        List<Document> packFdocs = new ArrayList<>();
+        if (Yes.equals(caseData.getDoYouNeedAWithoutNoticeHearing())) {
+            packFdocs.add(generateAccessCodeLetter(authorization, caseData, respondent, null, PRL_LET_ENG_FL401_RE2));
+        } else {
+            packFdocs.add(generateAccessCodeLetter(authorization, caseData, respondent, null, PRL_LET_ENG_FL401_RE3));
+        }
+        packFdocs.addAll(getNotificationPack(caseData, PrlAppsConstants.F, c100StaticDocs));
+        removeDuplicatesAndGetConsolidatedDocs(packEdocs, packFdocs, docs);
+        log.info("pack E docs {}", packEdocs);
+        log.info("pack F docs {}", packFdocs);
+        log.info("pack docs {}", docs);
+        if (ContactPreferences.email.equals(caseData.getApplicantsFL401().getContactPreferences())) {
+            checkAndSendEmailToDaApplicantLip(
+                caseData,
+                authorization,
+                emailNotificationDetails,
+                applicant,
+                docs,
+                packEdocs
+            );
+
+        } else {
+            sendSoaPacksToPartyViaPost(authorization, caseData, docs,
+                                       bulkPrintDetails,
+                                       element(caseData.getApplicantsFL401().getPartyId(), caseData.getApplicantsFL401()),
+                                       Templates.PRL_LET_ENG_AP1);
+        }
+
+        caseDataMap.put(UNSERVED_RESPONDENT_PACK, SoaPack.builder()
```

Comment:
This needs a discussion as we have to handle respondent citizen pack to be shown once they login.